### PR TITLE
post add-background-music to GCI

### PR DESCRIPTION
**Pull request by niccokunzmann**
### When you change [things like this in brackets] please remember to delete brackets after you change it.

# This pull request is related to issue(s), 
 #245

# The problem(s) I want to solve or The facilitie(s) I want to improve is/are,
allow students to claim the task

### My steps to solve it,
- add GCI link
- shorten so it fits 1500 characters

### CODE REVIEW

File: tasks/add-background-music.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Hints:
 - You can use the following open-source software:
   - [Audacity](https://www.audacityteam.org/) for recording
 - For your own recodrings, we recommend choosing the license `CC-BY` or `CC-BY-SA` if you like others to remix or `CC-BY-ND` if you do not like people editing your song. Choose a license here: https://creativecommons.org/choose/ Please do not choose NC as this might disable it for GSoC.
-- If someone wants to work with you, let them do that. You can use techniques like [pair programming](https://www.youtube.com/watch?v=vgkahOzFH2Q) or collaborate together on branches of your forks. You can attribute a pull-request also to other people who helped you out or submit a pull-request where you and an other person contributed together.
```
Comment (abishekvashok) [inline tasks/add-background-music.md]: Can we have a full stop pls?

Comment (niccokunzmann) [reply tasks/add-background-music.md]: ok, now I see... sure!

**Comment by niccokunzmann** [comment]
@abishekvashok Is this in your intention?

**Comment by niccokunzmann** [comment]
Thanks for the review!

**Pull request merged**